packages/playground/apps/default/specs-examples/ai/provider.ts

```diff
@@ -4,7 +4,7 @@ import { AIProvider } from '@blocksuite/presets';
 import { textToTextStream } from './request';
 
 export function setupAIProvider() {
-  AIProvider.provideAction('chat', options => {
+  AIProvider.provide('chat', options => {
     assertExists(options.stream);
     return textToTextStream({
```

Comment:
chat should use model `debug:chat:gpt4`

Reply:
This file is in fact no longer working and these changes are only to fix type issues.
See https://github.com/toeverything/AFFiNE/pull/6546/files#diff-e8a9e3cb567d0cc9ab5e33a5ad6cba230be641ad804f7b8440c852fcdbb98333R47 the related file change in affine